loadbalancer/roundrobin.go

```diff
@@ -121,7 +127,7 @@ func (rr *RoundRobinLoadBalancer) changeEndpointStatus(connectAddress string, st
 				break
 			}
 		} else {
-			panic("Non-LoadBalancingEndpoint in round robin router endpoint collection")
+			log.Error("Round robin load balancer misconfiguration: non round robin load balancer in round robin pool")
```

Comment:
After thinking about I made the two consistent by logging the error and continuing to iterate looking for an item in the data structure of the correct type. Note the only way an incorrect type could be present in the structure would be if someone modified post-boot and config processing the internal list of load balancers with the incorrect type.